friendships.go

```diff
@@ -0,0 +1,241 @@
+package twitter
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+// NoRetweetIDs calls the Twitter /friendships/no_retweets/ids.json endpoint.
+func (c *Client) NoRetweetIDs(ctx context.Context, stringifyIDs bool) (*UserIDsResponse, error) {
+	values := url.Values{}
+	values.Set("stringify_ids", strconv.FormatBool(stringifyIDs))
```

Comment:
Yes, but in this case, having `"stringify_ids"` set to true will result in a JSON parsing error (as we are unmarshaling to strings). So I guess the options are:
- set this permanantely to `true` and expect IDs as strings
- allow the user to set this value and unmarshal the JSON array of IDs into an `[]interface{}` (eww)
- use 64 bit integers with the JSON struct tag `string` so that unmarshaling will succeed, but then we've already made the decision that the ID will be an integer, making the `"stringify_ids"` option useless anyways.

To not make the response ugly, I think we have to choose whether IDs are strings or integers, which then makes `"stringify_ids"` an implementation detail. Elsewhere in this library, we've been using strings as Twitter recommends. What do you think?